index.rst

```diff
@@ -320,6 +320,13 @@ Welcome to PyTorch Tutorials
    :link: beginner/hyperparameter_tuning_tutorial.html
    :tags: Model-Optimization,Best-Practice
 
+.. customcarditem::
+   :header: Parametrizations Tutorial
+   :card_description: Learn how to use torch.nn.utils.parametrizations to put constriants on your parameters (e.g. make them orthogonal, symmetric positive definite, low-rank...)
```

Comment:
It's actually `nn.utils.parametrize` no?

Reply:
ooopsss!